reckless_cmd/src/reckless/cmd.rs

```diff
@@ -19,7 +19,7 @@ pub struct RecklessArgs {
 pub enum RecklessCommand {
     /// Install a single or a list of plugins.
     #[clap(arg_required_else_help = true)]
-    Install { plugin: Vec<String> },
+    Install { plugin_str: Vec<String> },
```

Comment:
```suggestion
    Install { plugin_str: Vec<String> },
```

why this name? this should be also the name that it is in the command line if you type `cargo run -- --help`